app/utils/galaxy-api/entities.ts

```diff
@@ -32,7 +33,20 @@ export interface WorkflowUrlParameter {
   url: string;
 }
 
-// Narrow type specific to the one kind of collection we use -- might be worth defining a more general collection type if we need other kinds of collections
+// Narrow type specific to the two kinds of collections we use -- might be worth defining a more general collection type if we need other kinds of collections
```

Comment:
the relationship between these two things could be clarified. meaning, a lot of places were using single vs paired language but not here, and technically the paired one is also a list no? it leaves the names of the interfaces alone slightly confusing, until you think about it from galaxy perspective instead of just the context of this pr. i understood on my second read through, but still, even if all we do is improve the comment above.. future danielle's brain would appreciate it